packages/sveltekit/README.md

```diff
@@ -117,9 +117,9 @@ export const load: LayoutLoad = async (event) => {
 
 This results in less server calls as the client manages the session on it´s own.
 
-### Typings
+### Typings with v2 from Supabase CLI
 
-In order to get the most out of TypeScript and it´s intellisense, you should import our types into the `app.d.ts` type definition file that comes with your SvelteKit project.
+In order to get the most out of TypeScript and it´s intellisense, you should import our types into the `app.d.ts` type definition file that comes with your SvelteKit project, where `import('./DatabaseDefinitions')` points to the generated types file outlined in [v2 docs here](https://supabase.com/docs/reference/javascript/release-notes#typescript-support) after you have logged in, linked, and generated types through the Supabase CLI.
```

Comment:
We should get rid of `import our types` since we don´t export any types for that use case.
Something like `import the generated Database types`